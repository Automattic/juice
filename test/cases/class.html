<html><body>
<div class="woot">Test</div>
<div class="woot">Test</div>
</body></html>

