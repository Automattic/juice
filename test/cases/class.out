<html><body>
<div class="woot" style="overflow: hidden;">Test</div>
<div class="woot" style="overflow: hidden;">Test</div>
</body></html>
